android/app/src/main/java/com/tourneynizer/tourneynizer/MainActivity.java

```diff
@@ -0,0 +1,28 @@
+package com.tourneynizer.tourneynizer;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.text.Editable;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        Button loginButton = findViewById(R.id.loginButton);
+        loginButton.setOnClickListener(new View.OnClickListener() {
+            @Override

```

Comment:
Can we use lambda functions here?

Reply:
Don't think we can under current API level.